### PR TITLE
Updated intro to CRUD lesson

### PR DESCRIPTION
**Pull request by s-espinosa**
@case-eee Updated CRUD lesson to go with the revised Task Master for discussion.

### CODE REVIEW

File: ruby_02-web_applications_with_ruby/outlines/intro_to_crud_redux.markdown [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+### Repository
+
+* If you have been following the [TaskManager](https://github.com/s-espinosa/task_manager_redux) tutorials up to this point, you should be able to use the version you already have.
+* If not, clone the [TaskManager Repo](https://github.com/s-espinosa/task_manager_redux), and switch to the crud-lesson branch.
+
+### Warm Up
+
+* What does CRUD stand for and why is it important?
+* What does MVC stand for? Create a diagram of the MVC pattern.
+* What part of the MVC does `task_manager_app.rb` represent in our current TaskManager app? What about `task.rb`?
```
Comment (case-eee) [inline ruby_02-web_applications_with_ruby/outlines/intro_to_crud_redux.markdown]: 👍

File: ruby_02-web_applications_with_ruby/outlines/intro_to_crud_redux.markdown [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+
+### CRUD Introduction
+
+Assuming we want to create full CRUD functionality in our Sinatra app for users accessing our site through a browser, there are seven routes that we will need to define. Let's make a chart together.
+
+(Here's a [completed chart](https://www.dropbox.com/s/vx3ocfsusjdrgfw/crud_in_sinatra.pdf?dl=0), but don't open it until you're finished!)
+
+### Code-Along
+
+We're going to follow the MVC design pattern (Rails uses this by default, but in Sinatra we will need to create this structure ourselves).
```
Comment (case-eee) [inline ruby_02-web_applications_with_ruby/outlines/intro_to_crud_redux.markdown]: Maybe something like?

> We're going to follow the MVC design pattern (Rails uses this by default, but in Sinatra we will need to create this structure ourselves) to implement the CRUD actions we're missing

File: ruby_02-web_applications_with_ruby/outlines/intro_to_crud_redux.markdown [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+    @task = Task.find(params[:id])
+    erb :edit
+  end
+```
+In our view, `edit.erb`:
+
+```erb
+<form action="/tasks/<%= @task.id %>" method="post">
+  <p>Edit</p>
+  <input type="hidden" name="_method" value="PUT" />
```
Comment (case-eee) [inline ruby_02-web_applications_with_ruby/outlines/intro_to_crud_redux.markdown]: Should we make a comment about why this is important?

File: ruby_02-web_applications_with_ruby/outlines/intro_to_crud_redux.markdown [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+    redirect "/tasks/#{id}"
+  end
+```
+
+In our TaskManager model:
+
+```ruby
+  def self.update(id, task_params)
+    database = SQLite3::Database.new('db/task_manager_development.db')
+    database.results_as_hash = true
```
Comment (case-eee) [inline ruby_02-web_applications_with_ruby/outlines/intro_to_crud_redux.markdown]: This, along with line 99 have been taken care of in `#initialize`, right? So we could just reference `@database` on line 101?

File: ruby_02-web_applications_with_ruby/outlines/intro_to_crud_redux.markdown [hunk 5]
```diff
@@ -0,0 +1,174 @@
...
+  end
+```
+
+### Work Time
+
+#### Mild: Skill Inventory
+
+Fork [this repository](https://github.com/turingschool-examples/skill-inventory-crud-redux) and CRUD out a skill inventory. Users should be able to enter a skill (create), see a list of all of the skills, see each skill individually (read), edit a skill (update), and delete a skill (delete).
+
+#### Spicy: Robot World
```
Comment (case-eee) [inline ruby_02-web_applications_with_ruby/outlines/intro_to_crud_redux.markdown]: What makes this version more difficult? The dashboard piece? When thinking of "robots" vs "skills", robots are so much cooler. I wonder what we could do to differentiate these two more? Maybe this is a for later thought, but I want to mention it here to document it somewhere!

**Comment by case-eee** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;0m3meFvL&#x2F;15-class-update-crud-lesson-to-use-new-task-manager"> [Class] Update CRUD lesson to use new Task Manager</a>

File: ruby_02-web_applications_with_ruby/outlines/intro_to_crud_redux.markdown (hunk 2 above)
Comment (s-espinosa) [inline ruby_02-web_applications_with_ruby/outlines/intro_to_crud_redux.markdown]: 👍

File: ruby_02-web_applications_with_ruby/outlines/intro_to_crud_redux.markdown (hunk 3 above)
Comment (s-espinosa) [inline ruby_02-web_applications_with_ruby/outlines/intro_to_crud_redux.markdown]: Agreed. Let me know what you think about the new language.

File: ruby_02-web_applications_with_ruby/outlines/intro_to_crud_redux.markdown (hunk 4 above)
Comment (s-espinosa) [inline ruby_02-web_applications_with_ruby/outlines/intro_to_crud_redux.markdown]: Left a note in the Task Manager repo. Let's discuss!

File: ruby_02-web_applications_with_ruby/outlines/intro_to_crud_redux.markdown (hunk 5 above)
Comment (s-espinosa) [inline ruby_02-web_applications_with_ruby/outlines/intro_to_crud_redux.markdown]: I think both the dashboard and the lack of a starter repo make this a little more difficult. Most students chose this, but didn't get the dashboard fully up and working last time through. I'm adding a separate card to the Trello board to track the idea of differentiating these some more.

**Pull request merged**